gnomad/assessment/validate_vep.py

```diff
@@ -0,0 +1,179 @@
+"""script to validate VEP annotation on protein-coding genes."""
```

Comment:
Typically `gnomad_methods` only contains functions that can be used within other repos that apply those functions to a dataset, so I would change this so that only `count_variant_per_interval` is in `gnomad_methods` move the actual calls of it to `gnomad_qc`.

Reply:
where should I put the actual calls in gnomad_qc?